### ISSUE TITLE
Commercial use - ready? Opinions

### ISSUE THREAD

**Issue by amocarski** [lusaxweb/vuesax#933]
Hi there! First of all congrats on making such a beautiful and stand out library!

Have one question for developers that are currently using this lib in any commercial use, does this lib is worth of give it a try in such a projects in terms of stability and stuff? I'd be grateful for having some IRL thoughts 😘

**Comment by racs4** [comment]
I think that this library has no support anymore, none of the issues are properly responded and a lot of PR with great solutions not approved.

**Comment by samhk222** [comment]
> does this lib is worth of give it a try in such a projects in terms of stability and stuff

No. I spent more time here at the issues section than the documentation, reading the hacks of fellow programmers.

**Comment by Tofandel** [comment]
No it's definitely not, just check [my fork](https://github.com/Tofandel/vuesax) and you'll understand just how much work was needed to make this production worthy, there is critical bugs in 70% of the components and the code quality is terrible, the deployed PR are also never tested and always absolutely always introduce new bugs
